packages/design-system-dashboard-cli/src/codeowners.json

```diff
@@ -0,0 +1,74 @@
+{
+  "*": ["va-platform-cop-frontend"],
+  "src/applications/_mock-form": ["platform-design-system-fe"],
+  "src/applications/burial-poc-v6": ["platform-design-system-fe"],
```

Comment:
Not blocking but I just want to call out that the [CODEOWNERS file gets updated regularly](https://github.com/department-of-veterans-affairs/vets-website/commits/main/.github/CODEOWNERS) so if these are hardcoded based on that reference, we might just need to keeps tabs on things a little bit or not count on the output as always being completely accurate.

Reply:
Good to note.